### ISSUE TITLE
incorrect predictions

### ISSUE THREAD

**Issue by kksnamboothiri** [amitnavindgi/sentiment-analysis-rnn#1]
every input i am getting Negative result

**Comment by amitnavindgi** [comment]
@kksnamboothiri Can you please share the classwise distribution of your dataset? This model may be overfitting your data.

**Comment by kksnamboothiri** [comment]
Without changing any code, trained your dataset given under data folder. Using tensorflow 1.3.1(1.6 version not available for me ).  
Environment: 
1.	visual studio 2017
2.	Python 3.6
3.	Windows 10
4.	tensorflow 1.3.1
Thanks for your replay

**Comment by amitnavindgi** [comment]
Yes, the dataset under the data folder is highly biased. Please check the readme.
